ipykernel/kernelbase.py

```diff
@@ -5,44 +5,49 @@
 
 import asyncio
 import concurrent.futures
-from datetime import datetime
-from functools import partial
+import inspect
 import itertools
 import logging
-import inspect
 import os
-from signal import signal, default_int_handler, SIGINT
-import sys
 import socket
+import sys
 import time
 import uuid
 import warnings
+from datetime import datetime
+from functools import partial
+from signal import SIGINT, SIGTERM, Signals, default_int_handler, signal
+
+if sys.platform == "win32":
```

Comment:
Should be `!=`?

Reply:
Yep, I'm dumb, and there are other issues on windows. I think I'm going to just not try to do that on windows, it might not even be a problem. I don't know how processes on windows work and maybe they get reaped anyway ?